specification/communication/data-plane/JobRouter/preview/2022-07-18-preview/communicationservicejobrouter.json

```diff
@@ -1185,6 +1225,14 @@
           },
           {
             "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "declineJobOfferRequest",
+            "description": "Request model for declining offer",
+            "schema": {
+              "$ref": "#/definitions/DeclineJobOfferRequest"
+            }
```

Comment:
Is the request body truly optional? What is the behavior if it is not provided?

Reply:
Same as before. For this particular route of decline offer, there is an optional property that `DeclineJobOfferRequest` wraps: `DateTimeOffset? RetryOfferAt`. If an empty body is provided, then internally, it would mean that the worker that declined the offer will not get any further offers for that particular job. If however, Contoso sets a value of `retryOfferAt`, then the worker will again get an offer for the same job after the given timestamp.